app/helpers/school_moves_helper.rb

```diff
@@ -13,4 +13,9 @@ def school_move_source(school_move)
 
     "#{source} updated"
   end
+
+  def school_move_warning_text(school_move)
```

Comment:
It looks like this helper is only used in one place now, I wonder if we need it as a helper or if it can go straight in to the view?